keystorage/aws/src/main/java/tech/pegasys/signers/aws/AwsSecretsManager.java

```diff
@@ -68,6 +80,39 @@ public Optional<String> fetchSecret(final String secretName) {
     }
   }
 
+  private Collection<SecretListEntry> listSecrets(final AbstractMap<String, String> tags) {
+    final ListSecretsRequest.Builder listSecretsRequestBuilder = ListSecretsRequest.builder();
+    if (tags != null) {
+      final Filter keys = Filter.builder().key("tag-key").values(tags.keySet()).build();
+      final Filter values = Filter.builder().key("tag-value").values(tags.values()).build();
```

Comment:
What do we do if we only want to filter by tag key and not tag value? How is this represented? Would be better to just have two lists of tag keys and tag values we want to filter on?